src/gui/torrentcontentmodelitem.h

```diff
@@ -35,47 +35,51 @@
 #include <QVariant>
 
 namespace prio {
-enum FilePriority {IGNORED=0, NORMAL=1, HIGH=6, MAXIMUM=7, MIXED=-1};
+    enum FilePriority {IGNORED=0, NORMAL=1, HIGH=6, MAXIMUM=7, MIXED=-1};
 }
 
 class TorrentContentModelFolder;
 
-class TorrentContentModelItem {
+class TorrentContentModelItem
+{
 public:
-  enum TreeItemColumns {COL_NAME, COL_SIZE, COL_PROGRESS, COL_PRIO, COL_REMAINING, NB_COL};
-  enum ItemType { FileType, FolderType };
+    enum TreeItemColumns {COL_NAME, COL_SIZE, COL_PROGRESS, COL_PRIO, COL_REMAINING, COL_AVAILABILITY, NB_COL};
```

Comment:
Then don't touch it at all! 
Let me repeat: if you produce "Follow coding style" commit, you should completely fix coding style and not just indentation.